projects/eslint-config/plugins/portal/tests/lib/rules/no-metal-plugins.js

```diff
@@ -17,6 +17,15 @@ const ruleTester = new MultiTester(parserOptions);
 
 ruleTester.run('no-metal-plugins', rule, {
 	invalid: [
+		{
+			code: "import {Affix} from 'metal-affix';",
```

Comment:
While we're at it, I'd make all of these consistent because we currently have some that use the default export and some that use the named export; just go for the default export, avoiding the extra width and punctuation of the curlies.